src/sage/algebras/all.py

```diff
@@ -67,3 +67,5 @@
 lazy_import('sage.algebras.cluster_algebra', 'ClusterAlgebra')
 
 lazy_import('sage.algebras.yangian', 'Yangian')
+
+import sage.algebras.vertex_operators_catalog as vertex_operators
```

Comment:
```suggestion
lazy_import("sage.algebras", "vertex_operators_catalog", "vertex_operators")
```
so it is not imported at startup.